azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/setup/SetupFragment.kt

```diff
@@ -62,7 +62,8 @@ internal class SetupFragment :
         setupJoinCallButtonHolderView.start(
             viewLifecycleOwner,
             viewModel.joinCallButtonHolderViewModel,
-            networkManager
+            networkManager,
```

Comment:
Suspicious to see reference to Manager in fragment/close to the UI.

Reply:
We need the manager reference since ViewModel checks for the network unavailability. It has been there previously. I just added a comma and it comes up as a new addition in my PR.